src/nvim/popupmenu.c

```diff
@@ -1528,6 +1549,8 @@ void pum_make_popup(const char *path_name, int use_mouse_pos)
                                   : curwin->w_wcol);
     if (ui_has(kUIMultigrid)) {
       mouse_grid = curwin->w_grid.target->handle;
+      pum_win_row_offset = curwin->w_winrow;
+      pum_win_col_offset = curwin->w_wincol;
```

Comment:
Is this needed? Doesn't `pum_position_at_mouse()` already do this?

Reply:
I removed it. Guess I was not expecting `pum_position_at_mouse` to be called when running an ex command. All tests still pass.